remove percent when displaying energy/rage/focus

bidaumm opened the pull request:
In powerdyn / "Mana - Auto" mode removes the percentage if mpmax is < 101 (i.e a energy/rage/focus bar) which can be off by 1 digit at times and look confusing (i.e energy looks like "35 - 36%")
also spaces the energy num display two spaces to the right for easier identification due to smaller number vs mana and clipping against rested icon

bidaumm commented:
also spaces the energy display two spaces to the right for easier identification due to smaller number vs mana

shagu commented:
> "mpmax == 100" wouldn't return true

I don't understand why mpax == 100 wouldn't return true :thinking: 
I have never seen UnitManaMax returning 99 for rage/focus/energy.

shagu commented:
> also spaces the energy num display two spaces to the right for easier identification due to smaller number vs mana and clipping against rested icon

I don't see the reason. Do you have a screenshot?

bidaumm commented:
Replaced with mpax == 100  again and its working this time, I must have been doing something wrong.
Also removed the white spaces as per discord convo

bidaumm commented:
replaced mpmax==100 with  UnitPowerType(unitstr) > 0

shagu commented:
Thanks! Squashed and merged by this commit: https://github.com/shagu/pfUI/commit/bc4caea63c44d6c69399a16ec3a7284fd86ff472
